core/src/main/scala/quasar/physical/ts/TSConfig.scala

```diff
@@ -42,7 +44,7 @@ object TSConfig {
 
 sealed trait Auth extends Product with Serializable {
   def sanitized: Auth = this match {
-    case Auth.PrivateKey(_) => Auth.PrivateKey("<redacted>")
+    case Auth.PrivateKey(_, _) => Auth.PrivateKey("<redacted>", None)
```

Comment:
Should this be `None` or `Some("<redacted>")` in the event that the input `PrivateKey` is `Some` on the `passphrase`?